tests/cpu_bugs/ipmi_to_qemu.pm

```diff
@@ -27,28 +27,33 @@ use ipmi_backend_utils;
 use power_action_utils 'power_action';
 use testapi;
 use utils;
-use version_utils 'is_sle';
+use version_utils 'is_sle', 'get_sles_release';
 my $cliect_ini_url    = get_var('CLIENT_INI');
 my $webui_hostname    = get_var('WEBUI_HOSTNAME');
 my $nfs_hostname      = get_var('NFS_HOSTNAME');
 my $qemu_worker_class = get_var('QEMU_WORKER_CLASS');
 sub run {
     my $self = shift;
+    my $current_dist;
     script_run("systemctl disable apparmor.service");
     script_run("aa-teardown");
-    zypper_call('rr devel_languages_perl devel_openQA devel_openQA_SLE-12 devel_openQA_SLE-15');
-    if (is_sle(">=15")) {
-        zypper_call('ar http://download.opensuse.org/repositories/devel:/languages:/perl/SLE_15/devel:languages:perl.repo');
-        zypper_call('ar http://download.opensuse.org/repositories/devel:/openQA/SLE_15/devel:openQA.repo');
-        zypper_call('ar http://download.opensuse.org/repositories/devel:/openQA:/SLE-15/SLE_15/devel:openQA:SLE-15.repo');
-    } elsif (is_sle(">=12")) {
-        zypper_call('ar http://download.opensuse.org/repositories/devel:/languages:/perl/SLE_12_SP4/devel:languages:perl.repo');
-        zypper_call('ar http://download.opensuse.org/repositories/devel:/openQA/SLE_12_SP4/devel:openQA.repo');
-        zypper_call('ar http://download.opensuse.org/repositories/devel:/openQA:/SLE-12/SLE_12_SP4/devel:openQA:SLE-12.repo');
+    my ($sles_running_version, $sles_running_sp) = get_sles_release();
+    print "sles_running_version: $sles_running_version\n";
+    print "sles_running_sp: $sles_running_sp\n";
+    if ($sles_running_sp gt '0') {
+        $current_dist = sprintf("SLE_%s_SP%s", $sles_running_version, $sles_running_sp);
+    } else {
+        $current_dist = sprintf("SLE_%s", $sles_running_version);
     }
+    print "sles_running_version: $sles_running_version, sles_running_sp: $sles_running_sp, current_dist: $current_dist\n";
```

Comment:
First 2 lines print is a bit redundant, in anyway version and sp will be printed by the 3rd print.

Reply:
Thanks, removed useless print.